test/test_ops.py

```diff
@@ -2195,7 +2195,15 @@ def test_fake_crossref_backward_no_amp(self, device, dtype, op):
     @skipOps('TestFakeTensor', 'test_fake_crossref_backward_amp', fake_backward_xfails | fake_autocast_backward_xfails)
     def test_fake_crossref_backward_amp(self, device, dtype, op):
         self._test_fake_crossref_helper(device, dtype, op, torch.cuda.amp.autocast)
-
+    @ops([op for op in op_db if op.is_factory_method], allowed_dtypes=(torch.float,))
+    def test_strided_layout(self, device, dtype, op):
+        samples = op.sample_inputs(device, dtype)
+        for sample in samples:
+            default_result = op(sample.input, *sample.args, **sample.kwargs)
+            self.assertEqual(default_result.layout, torch.strided)
+            strided_result = op(sample.input.to(layout=torch.strided), *sample.args, **sample.kwargs)
```

Comment:
This is not what yo want to test. You want to pass the layout to the op, not to the function `to`.

Reply:
@lezcano Thanks for the feedback. Is this what you had in mind? I would appreciate it if you could review these changes.

```
@ops([op for op in op_db if op.is_factory_method], allowed_dtypes=OpDTypes.all)
def test_strided_layout(self, device, dtype, op):
    samples = op.sample_inputs(device, dtype)
    for sample in samples:
        result = op(sample.input, *sample.args, layout=torch.strided, **sample.kwargs)
        self.assertEqual(result.layout, torch.strided)
```

Reply:
no need to pass allowed_dtypes. Also, this would not work if `sample.kwargs` already has `layout` in it, so yeah, almost like this, but not quite

Reply:
@lezcano I updated the test to be generic over all factory ops, pass layout to the op call directly, and handle kwargs with layout per your feedback. Please let me know if it fits well now.